internal/kubernetes/operator/config_exporter.go

```diff
@@ -395,3 +402,33 @@ func (e *ConfigExporter) exportAtlasStreamProcessing(projectName string) ([]runt
 
 	return result, nil
 }
+
+func (e *ConfigExporter) exportAtlasFederatedAuth(projectName string) ([]runtime.Object, error) {
+	result := make([]runtime.Object, 0)
+	// Gets the FederationAuthSetting
+	federatedAuthentificationSetting, err := e.dataProvider.FederationSetting(&admin.GetFederationSettingsApiParams{OrgId: e.orgID})
+	if err != nil {
+		return nil, err
```

Comment:
I would add context on what was happening when this failed:
```suggestion
		return nil, fmt.Errorf("failed to retrieve federation settings: %w", err)
```